third-culture-software/bhima
# Allow default access to Stock Setting menu item for admin, superuser

Updates the migration file so that the new Stock Settings menu item is available by default for Admin and superuser without having to edit their permissions.

Tested on IMCK production database.

To test:

- download and install a production database (with migration, etc)
- yarn dev
- Observe that the "Stock Settings" item is visible under "Stock" menu.
